### PR TITLE
Proxy

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/445

### CODE REVIEW

**Review by Herklos — COMMENTED** [review]
Nice :+1: can it also be configured from config.json?

**Comment by GuillaumeDSM** [comment]
up with env var config

File: octobot_trading/constants.py [hunk 1]
```diff
@@ -66,7 +66,11 @@
 FAILED_NETWORK_REQUEST_RETRY_ATTEMPTS = 5
 DEFAULT_REQUEST_TIMEOUT = int(os.getenv("DEFAULT_REQUEST_TIMEOUT", "20000"))    # default ccxt is 10s, use 20
 ENABLE_EXCHANGE_HTTP_PROXY_FROM_ENV = os_util.parse_boolean_environment_var(
-    "ENABLE_EXCHANGE_HTTP_PROXY_FROM_ENV", "True")
+    "ENABLE_EXCHANGE_HTTP_PROXY_FROM_ENV", "True"
+)
+EXCHANGE_HTTP_PROXY_AUTHENTICATED_URL = os.getenv("EXCHANGE_HTTP_PROXY_AUTHENTICATED_URL")
+EXCHANGE_HTTPS_PROXY_AUTHENTICATED_URL = os.getenv("EXCHANGE_HTTPS_PROXY_AUTHENTICATED_URL")
+EXCHANGE_SOCKS_PROXY_AUTHENTICATED_URL = os.getenv("EXCHANGE_SOCKS_PROXY_AUTHENTICATED_URL")
```
Comment (Herklos) [inline octobot_trading/constants.py:73]: :+1:

**Review by Herklos — APPROVED** [review]
:+1:

**Pull request merged**